libs/component-sample/src/lib/feature/search-result-list-sample/search-result-list-sample-data.component.ts

```diff
@@ -9,6 +9,19 @@ export class SearchResultListSampleDataComponent {
 
   constructor(private change: ChangeDetectorRef) { }
 
-@Input() model;
+  @Input() model;
 
-}
\ No newline at end of file
+  menu = {
+    trigger: 'primary',
+    actions: [
+      { id: 'DownloadBtn', icon: 'bars', text: 'Download' },
+      { id: 'FollowBtn', icon: 'bars', text: 'Follow' },
+      { id: 'ShareBtn', icon: 'bars', text: 'Share' }
+    ]
+  };
+
+  log(value) {
+    console.log(`%cLog: ${value}`, 'color: blue; font-weight: bold');
```

Comment:
What is the value being logged and since this is in the restultlist how would this work out side of this design

Reply:
I've added a simple log example that combines the action menu item getting clicked along with the current result id.